scripts/publish.js

```diff
@@ -75,25 +69,32 @@ async function run() {
     );
 
     // 2. Determine the appropriate npm tag to use
-    let tag = semver.prerelease(version) == null ? "latest" : "next";
+    let tag = version.includes("experimental")
+      ? "experimental"
+      : semver.prerelease(version) == null
+      ? "latest"
+      : "pre";
```

Comment:
Introduce an `experimental` tag and change from `next -> pre` to match what we're using nowadays